app/assets/javascripts/mumuki_laboratory/application/tooltip.js

```diff
@@ -1,3 +1,5 @@
 mumuki.load(() => {
-    $('[title]').tooltip();
+  [].slice.call(document.querySelectorAll('[data-bs-toggle="tooltip"]')).map(function (tooltipTriggerElement) {
```

Comment:
This code is pretty similar to [tooltip.js](https://github.com/mumuki/mumuki-laboratory/pull/1600/files#diff-84fdbab3831f91cea97dbff2533446a92104c070a9e1925b079d1c666c3d6af1R2). Is it worth extracting this code to a function? I don't know if we would repeat this in the future

Reply:
It is similar because both need initializing. To be honest, this is straight from the Bootstrap docs. If you ask me about extracting, I would say no, not really - there's nothing else we could have to initialize, so there shouldn't be more cases like this.

If you don't like the `[].slice.call` part or something, else, though, I'm open to suggestions :-)

Reply:
I think the return is not necessary in the map function because you don't do anything with the list resulting from the mapping.
If I'm not mistaken, it can be done directly using forEach, but you will know that better than I do.

```javascript
document.querySelectorAll('[data-bs-toggle="tooltip"]').forEach((tooltip) => new bootstrap.Tooltip(tooltip))
```

Reply:
This works! I've changed it for toasts as well.